main.py

```diff
@@ -1605,7 +1605,7 @@ def format_completion(self, item) -> 'Tuple[str, str]':
             insertText = item.get("insertText")
         if insertText[0] == '$':  # sublime needs leading '$' escaped.
             insertText = '\$' + insertText[1:]
-        return ["{}\t{}".format(label, detail), insertText]
+        return "{}\t{}".format(label, detail), insertText
```

Comment:
Not sure, whether it's to be handled by LSP or the pyls, but `detail` returns `def function` for a function completion, which makes completions look a bit too verbose:

function    _def function_

Reply:
This would be something for pyls to look at.